### PR TITLE
Move Extension Localization Information from Extbase Fluid Book

### PR DESCRIPTION
**Pull request by linawolf**
Resolves https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues/720

releases: main, 11.5

### CODE REVIEW

File: Documentation/ExtensionArchitecture/HowTo/Localization/Fluid.rst [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+   Fluid Templates; Multi-language
+   Localization; Fluid templates
+.. _extension-localization-fluid:
+
+==============================
+Multi-language Fluid templates
+==============================
+
+.. tip::
+   Texts can also be translated direclty in the PHP code, for example in
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/HowTo/Localization/Fluid.rst:12]:
```suggestion
   Texts can also be translated directly in the PHP code, for example in
```

File: Documentation/ExtensionArchitecture/HowTo/Localization/Fluid.rst [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+
+.. code-block:: html
+   :caption: EXT:my_extension/Resources/Private/Templates/SomeTemplate.html
+
+   <f:translate key="LLL:EXT:my_extension/Resources/Private/Language/yourFile.xlf:yourKey" />
+   <!-- or as inline Fluid: -->
+   {f:translate(key: 'LLL:EXT:my_extension/Resources/Private/Language/yourFile.xlf:yourKey')}
+
+
+The text fragment will now me output in the current frontend language
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/HowTo/Localization/Fluid.rst:60]:
```suggestion
The text fragment will now be displayed in the current frontend language
```

File: Documentation/ExtensionArchitecture/HowTo/Localization/Fluid.rst [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+defined in the site configuration if the translation file of the requested
+langauge can be found in the location of the prefix.
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/HowTo/Localization/Fluid.rst:62]:
```suggestion
language can be found in the location of the prefix.
```

File: Documentation/ExtensionArchitecture/HowTo/Localization/Fluid.rst [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+.. code-block:: html
+   :caption: EXT:my_extension/Resources/Private/Templates/SomeTemplate.html
+
+   <f:translate key="LLL:EXT:my_extension/Resources/Private/Language/yourFile.xlf:yourKey" />
+   <!-- or as inline Fluid: -->
+   {f:translate(key: 'LLL:EXT:my_extension/Resources/Private/Language/yourFile.xlf:yourKey')}
+
+
+The text fragment will now me output in the current frontend language
+defined in the site configuration if the translation file of the requested
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/HowTo/Localization/Fluid.rst:61]:
```suggestion
defined in the site configuration, if the translation file of the requested
```

**Review by brotkrueml — COMMENTED** [review]
For me it looks like different patches are mixed up here now, which have nothing in common with localization.

**Comment by linawolf** [comment]
yeah, this must have happend oppon rebasing. let me try to cherry pick this freshly

**Comment by linawolf** [comment]
Cherry picked freshly into https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/pull/2093

**Pull request closed without merging**